### ISSUE TITLE
[🐛 Bug]: Document for "Locator strategies" does explain what is "By" 

### ISSUE THREAD

**Issue by HGtz2222** [SeleniumHQ/seleniumhq.github.io#1393]
### What happened?

https://www.selenium.dev/documentation/webdriver/elements/locators/

In this page, the demo code show how to locator an element, such as:

`driver.find_element(By.CLASS_NAME, "information")`

But I cannot see what is the "By", and how can I import "By" in my code. 

If I use Java, IDE can help me generate code to import "By" . But if I use Python, most IDE cannot do such thing. 
So I hope the document can add the import statement for these key identifiers such as "By". 

Thanks very much! 

### What browsers and operating systems are you seeing the problem on?

uncorrelated

**Comment by Grs257** [comment]
You can debug or enquire it on open source

**Comment by titusfortner** [comment]
So, first, we are trying to move away from static code, and move all examples to our repo.
But even in our repo we are trying to only import the lines of code that matter. At least this way there will be a big link to see the full example.

At the same time, I'm leaning towards us just using the full path for classes instead of doing imports to make it more obvious where it is coming from.
e.g.: 
```
webdriver.common.by.By.ID
```

**Comment by eaccmk** [comment]
So do we thing this is a bug and need a fix or CR ?

@HGtz2222  I see for Python you can import `By` using 

```python
from selenium.webdriver.common.by import By
```

**Comment by titusfortner** [comment]
Yes, we should either do the full path for it, or include the import line in a second reference.

**Comment by diemol** [comment]
Closed via #1880
